src/main.rs

```diff
@@ -102,6 +105,16 @@ fn settings() -> io::Result<Settings> {
                     DEFAULT_THRESHOLD.code_str(),
                 )),
         )
+        .arg(
+            Arg::with_name(KEEP_OPTION)
+                .value_name("KEEP")
+                .short("k")
+                .long(KEEP_OPTION)
+                .multiple(true)
+                .number_of_values(1)
```

Comment:
Is this not the default when `value_name` is specified? What happens if this is omitted?

Reply:
If you remove this, you can actually specify multiple with the same flag. So with my latest change you can now do:

```
--keep pattern1 pattern2 pattern3 --keep pattern4 pattern5
```

Reply:
Oh I see—thanks for explaining it. Actually, would you mind changing it back? I'm worried about how this will interact with other flags that might be added in the future. Having one pattern per flag seems like a safer choice.